packages/apps-config/src/api/params/proposalThresholds.ts

```diff
@@ -1,38 +1,36 @@
 // Copyright 2017-2021 @polkadot/app-config authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import type { ApiPromise } from '@polkadot/api';
-
 import { KULUPU_GENESIS, KUSAMA_GENESIS, POLKADOT_GENESIS } from '../constants';
 
-const PROPOSE_THRESHOLDS: Record<string, number> = {
-  [KULUPU_GENESIS]: 1,
-  [KUSAMA_GENESIS]: 0.5,
-  [POLKADOT_GENESIS]: 0.6,
-  default: 0.5
-};
+export interface Threshold {
+  value: number;
+  option: 'AtLeast' | 'MoreThan';
```

Comment:
does it make sense to create an enum for these options? for simplifying further maintaining and exclude typos